driftbase/api/events.py

```diff
@@ -44,17 +43,20 @@ def post(self):
         """
         required_keys = ["event_name", "timestamp"]
 
-        verify_log_request(request, required_keys)
+        if request.headers.get('Content-Encoding', '') == 'gzip':
+            events = json.loads(gzip.decompress(request.data))
```

Comment:
Maybe add a try/except here to at least guard against 500 error and send a 400 error instead with a message indicating the payload is malformed?